tests/epioncho_tests/test_tools.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+
+from epioncho_ibm.tools import Data, flatten_and_sort
+
+
+def test_flatten_and_sort_empty_input():
+    assert [] == flatten_and_sort([])
+
+
+def test_flatten_and_sort_single_empty_row():
+    with pytest.raises(AttributeError):
+        flatten_and_sort([[]])
+
+
+def test_flatten_and_sort_single_data_multiple_keys():
+    inputData: list[Data] = [
+        {
+            (2000, 0, 80, "test_measure"): 1,
+            (2000, 0, 80, "test_measure_2"): 1,
+            (2001, 0, 80, "test_measure"): 1,
+        }
+    ]
+    outputData = flatten_and_sort(inputData)
+    assert len(outputData) == 3
+    assert outputData[0] == (2000, 0, 80, "test_measure", 1)
+    assert outputData[1] == (2000, 0, 80, "test_measure_2", 1)
+    assert outputData[2] == (2001, 0, 80, "test_measure", 1)
+
+
+def test_flatten_and_sort_multiple_data_single_key():
+    inputData: list[Data] = [
+        {(2000, 0, 80, "test_measure"): 1, (2001, 0, 80, "test_measure"): 1},
+        {(2000, 0, 80, "test_measure"): 2, (2001, 0, 80, "test_measure"): 2},
```

Comment:
⛏️ You might use 3, 4 to be sure it is slicing the right way!  
```suggestion
        {(2000, 0, 80, "test_measure"): 3, (2001, 0, 80, "test_measure"): 4},
```